exe/main.cc

```diff
@@ -27,7 +27,7 @@ int startAgent(std::vector<std::string> arguments) {
     std::unique_ptr<Util::PIDFile> pidf_ptr;
 
     try {
-        if (!Configuration::Instance().get<bool>("foreground")) {
+        if (!HW::GetFlag<bool>("foreground")) {
```

Comment:
It would be nice to have the usual comment about using HW because Configuration may not be initialized.